### PR TITLE
feat: adds a mentorship program section in about

### LINKED ISSUE #3029: feat: Program section in about us Page
**Is your feature request related to a problem? Please describe.**
A program section will contain the open-source programs in which CircuitVerse participated (Past and Present).

**Are you working on this? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by vikaskumar2703**
Fixes #3029 
#### Describe the changes you have made in this PR -
this pr adds a mentorship program section in about page of circuit verse main websites containing names and timeline of programs in which circuit verse has taken part as an organisation till date.
### Screenshots of the changes (If any) -

Capture

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]
@vikaskumar2703 fix the codeclimate and hound issues!

File: app/views/about/index.html.erb [hunk 1]
```diff
@@ -107,6 +107,45 @@
...
+          <p class=" fw-bolder mentorship-program-2"><%= t("about.mentorship_program_2") %></p>
+          <p class="mentorship_program_time"><%= t("about.mentorship_program_2_timeline") %></p>
+          </div>
+          </div>
+          <div class="card mentorship-program-card"> 
+          <%= image_tag "SVGs/gci-logo.svg", height: "100", width: "50", alt: "Email Icon", class: "card-img-top" %>
+          <div class="card-body">
+          <p class=" fw-bolder mentorship-program-3"><%= t("about.mentorship_program_3") %></p>
+          <p class="mentorship_program_time"><%= t("about.mentorship_program_3_timeline") %></p>
+          </div>
```
Comment (tachyons) [inline app/views/about/index.html.erb:136]: Could you indent the html ?
```suggestion
          <div class="card-body">
            <p class=" fw-bolder mentorship-program-3"><%= t("about.mentorship_program_3") %></p>
            <p class="mentorship_program_time"><%= t("about.mentorship_program_3_timeline") %></p>
          </div>
```

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]
@tachyons what if we place this section after core-team section or above it?

**Comment by vikaskumar2703** [comment]
> @tachyons what if we place this section after core-team section or above it?

above would be better option

**Comment by tachyons** [comment]
> @tachyons what if we place this section after core-team section or above it?

Below the teams sections

File: config/locales/views/about/en.yml [hunk 2]
```diff
@@ -18,3 +18,13 @@ en:
     contributors_heading: "CONTRIBUTORS"
     contributors_description: "Meet the awesome people of CircuitVerse community that've made this platform what it is now."
     become_contributor: "Become a Contributor"
+    mentorship_programs_heading: "MENTORSHIP PROGRAMS" 
+    mentorship_programs_description: "CircuitVerse has been participating in various mentoring programs, including:"
+    program_gsoc: "Google Summer of Code "
+    program_gsoc_timeline: "2020, 2021, 2022, 2023"
+    program_gsod: "Google Season of Docs "
+    program_gsod_timeline: "2023"
```
Comment (vedant-jain03) [inline config/locales/views/about/en.yml:26]:
```suggestion
    program_gsod_timeline: "2020"
```

File: config/locales/views/about/en.yml [hunk 3]
```diff
@@ -18,3 +18,13 @@ en:
     contributors_heading: "CONTRIBUTORS"
     contributors_description: "Meet the awesome people of CircuitVerse community that've made this platform what it is now."
     become_contributor: "Become a Contributor"
+    mentorship_programs_heading: "MENTORSHIP PROGRAMS"
```
Comment (vedant-jain03) [inline config/locales/views/about/en.yml:21]: Add hindi translation as well, here: `config/locales/views/about/hi.yml`

File: config/locales/views/about/en.yml [hunk 4]
```diff
@@ -18,3 +18,13 @@ en:
     contributors_heading: "CONTRIBUTORS"
     contributors_description: "Meet the awesome people of CircuitVerse community that've made this platform what it is now."
     become_contributor: "Become a Contributor"
+    mentorship_programs_heading: "MENTORSHIP PROGRAMS" 
+    mentorship_programs_description: "CircuitVerse has been participating in various mentoring programs, including:"
+    program_gsoc: "Google Summer of Code "
+    program_gsoc_timeline: "2020, 2021, 2022, 2023"
```
Comment (vedant-jain03) [inline config/locales/views/about/en.yml:24]: Make the years embed with respective program circuitverse page, for example : [2021](https://summerofcode.withgoogle.com/archive/2021/organizations/6293580166987776)

Note: you can make translation act as html by suffixing the key by `_html` for example `program_gsoc_timeline_html`

**Review by vedant-jain03 — COMMENTED** [review]
@vikaskumar2703 Add some followup changes
Also, make this section below teams!

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]
@vikaskumar2703 Any updates on changes?

**Comment by vikaskumar2703** [comment]
> @vikaskumar2703 Any updates on changes?

working on it . i have brought the mentorship section below core team but having some difficulty in translation and embeding the year thing.

**Comment by vikaskumar2703** [comment]
@vedant-jain03 
done with adding hindi translation as well 
only thing left is emedding years with links . can you pl share exactly what all links need to be embedded. That would be great help.

**Comment by vikaskumar2703** [comment]
> 
also what should be the most efficient way to embed the links because the idea in my mind is very complex

**Review by vedant-jain03 — APPROVED** [review]
@vikaskumar2703 Thanks for contribution, LGTM

cc @tachyons

**Comment by vikaskumar2703** [comment]
> @vikaskumar2703 Thanks for contribution, LGTM
> 
> cc @tachyons 

Pleasures all mine @vedant-jain03

**Pull request closed without merging**